website/docs/guides/sso/keycloak/configuring-oidc-with-keycloak.mdx

```diff
@@ -0,0 +1,367 @@
+
+## What is Keycloak?
+
+[Keycloak](https://www.keycloak.org/) is an identity service that uses [OpenID Connect](https://openid.net/connect/) to drive authentication for other apps. It has powerful integrations such as LDAP or Microsoft AD.
+
+Alternative solutions for identity and access management exist such as [Dex](https://dexidp.io/).
+
+## Deploying Keycloak
+
+### Creating a namespace for the Keycloak Deployment
+
+Create a namespace where Keycloak will be installed
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: auth
+```
+
+### Deploying Keycloak
+
+As we did before, we can use HelmRepository and HelmRelease objects to let Flux deploy everything.
+
+
+<details><summary>Expand to see resource manifests</summary>
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: bitnami
+spec:
+  url: https://charts.bitnami.com/bitnami
+  interval: 1m
+
+---
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: keycloak
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: keycloak
+      version: 17.1.1
+      sourceRef:
+        kind: HelmRepository
+        name: bitnami
+
+  values:
+    ## ================================================================================================
+    ## Keycloak App Settings
+    ## ================================================================================================
+    auth:
+      adminUser: admin
+      adminPassword: admin123  # <-- make use of `existingSecret` functionality to hide passwords
+
+    proxy: reencrypt
+    tls:
+      enabled: true
+      autoGenerated: true
+
+    replicaCount: 1
+
+    production: true
+
+    ## ================================================================================================
+    ## Keycloak ingress parameters
+    ## ================================================================================================
+    httpRelativePath: /keycloak/
+    ingress:
+      enabled: true
+      ingressClassName: nginx
+      hostname: auth.mydomain.com
+      path: /keycloak/
+      tls: true
+      annotations:
+        cert-manager.io/cluster-issuer: "letsencrypt-production"
+
+    ## ================================================================================================
+    ## PostgreSQL Configuration
+    ## ================================================================================================
+    postgresql:
+      enabled: true
+      architecture: standalone
+      auth:
+        username: bn_keycloak
+        database: bitnami_keycloak
+        password: potgres123  # <-- make use of `existingSecret` functionality to hide passwords
+        postgresPassword: postgres123  # <-- make use of `existingSecret` functionality to hide passwords
+```
+
+</details>
+
+### Configuring a new Keycloak Realm
```

Comment:
```suggestion
## Configuring a new Keycloak Realm
```

IMHO this guide should focus on how to configure keycloak for the specific use case of using it as an IdP.